docs/manual/src/tutorial/foreign_language_bindings.md

```diff
@@ -38,24 +38,48 @@ creating a binary for each crate that uses UniFFI.  You can avoid this by creati
 
 Then your can run `uniffi-bindgen` from any create in your project using `cargo run -p uniffi-bindgen [args]`
 
-## Running uniffi-bindgen
+## Running uniffi-bindgen with a single UDL file
+
+Use the `generate` command to generate bindings by specifying a UDL file.
 
 ### Kotlin
 
-Run
+From the `example/arithmetic` directory, run:
 ```
-cargo run --bin uniffi-bindgen generate src/math.udl --language kotlin
+cargo run --bin uniffi-bindgen generate src/arithmetic.udl --language kotlin
 ```
-then have a look at `src/uniffi/math/math.kt`
+then have a look at `src/uniffi/arithmetic/arithmetic.kt`
 
 ### Swift
 
 Run
 ```
-cargo run --bin uniffi-bindgen generate src/math.udl --language swift
+cargo run --bin uniffi-bindgen generate src/arithmetic.udl --language swift
 ```
-then check out `src/math.swift`
+then check out `src/arithmetic.swift`
```

Comment:
I noticed the path here and above at line 51 were different, so decided to actually try this command out - and it failed: `error: no bin target named uniffi-bindgen.` - which I guess is somewhat expected given the discussion above re uniffi-bindgen, but it still seems unfortunate - I wonder if we should get an issue on file for this? It seems a reasonable expectation that it's clear to new users exactly how to run example commands in our manual?

Reply:
Yeah, I actually added a binary named `uniffi`, but @badboy points out that would mean that we now distributed a binary again, which means it's much more likely that users would use it and get version mismatches.

Maybe this could be solved by naming?  If we called the binary `uniffi-internal-bindgen` or something that should hopefully discourage users from using it.  We can make it easy to call inside the workspace with local cargo configuration.

Reply:
We have [uniffi-bindgen-cli](https://github.com/mozilla/uniffi-rs/tree/main/examples/app/uniffi-bindgen-cli) as the internal binary for that, used in the sample app. Maybe we should just lift this up as a `uniffi-internal-bindgen` crate we don't publish?